openfl-tutorials/interactive_api/HPU/PyTorch_TinyImageNet/workspace/hpu_pytorch_tinyimagenet.ipynb

```diff
@@ -0,0 +1,509 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "26fdd9ed",
+   "metadata": {},
+   "source": [
+    "# Federated PyTorch TinyImageNet Tutorial"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "895288d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!pip install -r requirements.txt"
```

Comment:
This can be removed if you have validated without additional packages. To avoid confusion to the user.